python/pyproject.toml

```diff
@@ -0,0 +1,48 @@
+[tool.poetry]
+name = "graphframes-py"
+version = "0.8.4"
```

Comment:
Let's use something like this: https://pypi.org/project/poetry-dynamic-versioning/?